server/channels/api4/apitestlib_test.go

```diff
@@ -16,7 +16,7 @@ func TestEnvironmentVariableHandling(t *testing.T) {
 
 	originalConsoleLevel := os.Getenv("MM_LOGSETTINGS_CONSOLELEVEL")
 	defer func() {
-		// Restore original environment variables
+		// RestoreContentFlaggedPost original environment variables
```

Comment:
Again, seems like accidental replacement

Reply:
Fixed. Seems like the refactoring doesn't work that well with comments.